src/client/client.js

```diff
@@ -272,8 +272,6 @@ class _ClientImpl {
           server: multiplayer.server,
           socketOpts,
         });
-      } else if (multiplayer.transport !== undefined) {
-        this.transport = multiplayer.transport;
```

Comment:
Shall we just leave this option as `multiplayer.transport` and treat it as a constructor here?  I'd prefer to use `server` for the socketio stuff.

Basically I'm proposing changing this line to `this.transport = new (multiplayer.transport)({ ... })` and pass in everything that's passed in from the if branch right above.